metricflow-semantics/metricflow_semantics/specs/query_param_implementations.py

```diff
@@ -137,13 +136,16 @@ def query_resolver_input(  # noqa: D102
     ) -> ResolverInputForMetric:
         naming_scheme = MetricNamingScheme()
         return ResolverInputForMetric(
-            input_obj=self,
+            input_obj=self.with_alias(None),
```

Comment:
is there a need for calling `with_alias` here?

Reply:
Oops good call, removing all of those from this file!